data/css/library_list.scss

```diff
@@ -0,0 +1,66 @@
+@import 'mesh';
```

Comment:
@rmacqueen I get this error:

```
  SCSS     data/css/library_list.css
WARNING: 'arrow-symbolic.svg' was not found (or cannot be read) in ./images
WARNING: 'deckbg.svg' was not found (or cannot be read) in ./images
WARNING: 'pdf-background.png' was not found (or cannot be read) in ./images
WARNING: 'noise.png' was not found (or cannot be read) in ./images/../templates/images
```

This is because all top level themes should import the sass configuration files

```
@import "project-setup";
@import "compass";
```

Take a look what other top level themes do.